ehitchhiking-ui/src/app/shared/modals/profile-modal/profile-modal.component.sass

```diff
@@ -43,15 +71,15 @@
                     text-align: center
             &-title
                 width: 10%
+                margin-right: 0
                 @include flexbox(flex-start)
 
         .profile-add-car
-            position: absolute
-            right: $default-offset
+            width: 100%
             @include flexbox()
             .profile-add-car-button, .profile-close-car-button
                 position: fixed
-                margin-top: -1rem
+                margin-top: -0.5rem
```

Comment:
pls try to avoid using negative values in the future

Reply:
got it!